### DISCUSSION TITLE
Top margin has some value for the first page

### DISCUSSION THREAD

**Discussion by striveandsurvive** [gotenberg/gotenberg#927] [Q&A] [answered]
Hi,

I have set marginTop value to 0mm and it is observed working as expected for all the pages except the first page of the PDF. The first page has some space at the top though the margin is set to 0. This happens only for the first page.

It does not have any header/footer specified in the request.

Is this expected behavior? Any solution to fix this?

Thanks in advance.

**Comment by gulien** [comment, marked as answer]
Hello @striveandsurvive,

Might work with:

```
@page { size: A4 portrait; margin: 0; }
```

Alas, the margin feature is implemented directly in Chromium, so there is nothing much I can do on my side.

**Reply by striveandsurvive** [reply]
Thanks. I tried <body style="margin-top:0px"> and it fixed the issue.
